index.html

```diff
@@ -5,7 +5,8 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
-    <title>Dozer | Managed BOSH</title>
+    <title>Dozer | The GUI for BOSH</title>
+    <meta name="description" content="Dozer offers a graphical user interface (GUI) for BOSH that makes it easy to connect, monitor and manage multiple instances of BOSH." />
```

Comment:
To give you an idea, this is what Google will show people:

![screen shot 2016-09-20 at 12 01 09](https://cloud.githubusercontent.com/assets/1488816/18665962/e883edec-7f29-11e6-8965-f16c396912d0.png)